browser/resources/settings/brave_overrides/basic_page.ts

```diff
@@ -254,7 +240,10 @@ RegisterPolymerTemplateModifications({
       sectionHelpTips.setAttribute('is', 'dom-if')
       sectionHelpTips.setAttribute('restamp', true)
       sectionHelpTips.setAttribute('if', '[[showPage_(pageVisibility.braveHelpTips)]]')
-      sectionHelpTips.content.appendChild(createSectionElement(
+      sectionHelpTips.content.appendChild(createSectionElement(/* Copyright 2021 The Brave Authors. All rights reserved.
+      * This Source Code Form is subject to the terms of the Mozilla Public
+      * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+      * You can obtain one at http://mozilla.org/MPL/2.0/. */
```

Comment:
not really needed- but to be consistent with other places, can we change to https?

Reply:
Oops, it's accidently added. Deleted.